spec/processors/jpeg2k_spec.rb

```diff
@@ -54,4 +56,25 @@
       expect(r).to eq(described_class.calculate_recipe(args, 'grey', 7200))
     end
   end
+
+  describe "#encode" do
+    it "executes the external utility" do
+      expect(described_class).to receive(:execute) { 0 }
+      described_class.encode('infile', 'recipe', 'outfile')
+      # expect(Hydra::Derivatives::Processors::ShellBasedProcessor).to have_received(:execute)
+    end
+  end
+
+  describe "#tmp_file" do
+    it "returns a temp file with the correct extension" do
+      f = described_class.tmp_file('test')
+      expect(f).to match(/sufia.*test$/)
```

Comment:
That conversation was based on #99 .   I agree that it seems possible to remove the sufia from the regex and maybe change this to an ends_with? ".test".